rocAL/rocAL/include/meta_data/bounding_box_graph.h

```diff
@@ -25,13 +25,15 @@ THE SOFTWARE.
 #include "meta_data_graph.h"
 #include "meta_node.h"
 
```

Comment:
Add a comment here stating the use of union.

Reply:
Done